config/routes.rb

```diff
@@ -10,6 +10,10 @@
     patch '/shelters/:id', to: 'shelters#update'
     delete '/shelters/:id', to: 'shelters#destroy'
 
+    get '/shelters/:id/new_review', to: "reviews#new_review"
```

Comment:
any reason not to do `reviews#new`?

Reply:
It was just a consistency thing. Kept it all the same so it was easy to find.